plugins/plugin-gdb/che-plugin-gdb-server/src/main/java/org/eclipse/che/plugin/gdb/server/GdbDebugger.java

```diff
@@ -346,6 +347,9 @@ public void stepOut(StepOutAction action) throws DebuggerException {
   }
 
   @Override
+  public void runToLocation(RunToLocationAction action) throws DebuggerException {}
```

Comment:
No need to add implementation in all debuggers. Just add default method into the Debugge interface.

Reply:
done